_includes/sidenav.html

```diff
@@ -8,34 +8,27 @@
 
    Create nav markup when given:
 
-   items:               An array of tuples used to display nav items. 
-                        format: (href, id, caption)
+   items:               An array of tuples used to display nav items.
+                        format: (href, id, caption, children)
 
-   active_item_id:      A string indicating the id of the nav item 
+   active_item_id:      A string indicating the id of the nav item
                         to be activated.
 
-   additional_classes:  Extra classes you wish to add to the nav, space separated.
+   additional_classes:  Extra classes you wish to add to the nav,
+                        space separated.
+
+  @TODO: Document how second-level items are set up.
 
    ========================================================================== #}
 
 {% macro render(items, active_item_id, additional_classes) %}
 
 <nav class="nav-secondary expandable {{ additional_classes }}">
-    <h1 class="u-visually-hidden">
-{%- for href, id, caption in items %}
-    {% if id == 'index' %}
-        {{ caption|e }} navigation
-    {% endif %}
-{%- endfor %}
-    </h1>
-    <div class="expandable_header">
+    <h2 class="u-visually-hidden">Section navigation</h2>
+    <div class="expandable_header nav-secondary_header">
         <button class="expandable_target nav-secondary_link nav-secondary_link__button">
```

Comment:
We've updated this to remove over-writing the link styles https://github.com/cfpb/cfgov-refresh/pull/1144/files#diff-720db790d1102c4830e1cda59a69f018R28